pgcli/main.py

```diff
@@ -64,6 +64,9 @@
 
 from collections import namedtuple
 
+if sys.version_info[0] > 2:
+    unicode = str
```

Comment:
I think there is already a Python 2/3 check in `pgcli/encodingutils.py`.

So, you might consider doing what we do at mycli and CLI Helpers. In a different module, handle the Python 2 vs. 3 logic so that it doesn't end up in `main.py`. Then, you just import `text_type` (nothing magical about that variable name, it's just a popular name choice that many projects use). Here it is in mycli:
https://github.com/dbcli/mycli/blob/master/mycli/encodingutils.py#L9

I think it's good to stay away from using the standard library names (e.g. `unicode` and `str`) so as to prevent confusion about what class we're actually referencing.